.buildkite/benchmarks/pipeline.yml

```diff
@@ -105,6 +114,9 @@ steps:
         key: "gpu_amip_diagedmf"
         command: "srun julia --threads=3 --color=yes --project=experiments/ClimaEarth/ experiments/ClimaEarth/run_amip.jl --config_file $BENCHMARK_CONFIG_PATH/amip_diagedmf.yml --job_id gpu_amip_diagedmf"
         artifact_paths: "experiments/ClimaEarth/output/amip/gpu_amip_diagedmf_artifacts/*"
+        env:
+          CLIMACOMMS_CONTEXT: "MPI"
```

Comment:
These are actually single-threaded runs. You can tell because no additional `ntasks` are requested in the `agents`. So, you should remove `MPI` from here.

Reply:
I am not sure I get it - line 115 has `--threads=3` and line 123 has `slurm_ntasks: 4` ?

Reply:
Ah, you are right. The diff was cut off and I did see line 123. What you have is correct!